charts/sonarqube-dce/templates/application.yml

```diff
@@ -0,0 +1,58 @@
+{{- if .Values.gcp_marketplace }}  # declared in google-cloud-marketplace-k8s-app value folder
```

Comment:
For now, we use the gcp_marketplace value only in the schema.yaml to activate this manifest. However, shall we document it anyways in the README and values.yaml of the chart? This might be useful to not forget about this value and have all the values clearly documented...